policyengine_us/data/parameters/gov/aca/ca/gra.yaml

```
@@ -0,0 +1,165 @@
+description: ACA geographic rating areas for California.
```

```suggestion
description: California assigns counties to these Affordable Care Act geographic rating areas.
```
Example for other descriptions

---

The problem with your suggestion
```
California assigns counties to these Affordable Care Act geographic rating areas.
```
is that it is incorrect.

---

Have revised the description for both the `gra.yaml` and `slspc.yaml` parameter files.

---

DONE